packages/backend/src/index.ts

```diff
@@ -19,6 +21,12 @@ function start() {
   });
 
   app.setup(server);
+
+  const seededTags: Partial<NFCTag>[] = seedNFCTags;
```

Comment:
@JanEberlein As importing json seems a bit unstable / not properly configured you could also try directly putting the data into code. It wont be to bad as we will remove the sample data anyways as soon as we support creation via the app.

```ts
async function seed() {
  try {
    await app.service('nfc-readers').create({
      _id: '', // use a fixed id to prevent duplicates
      owner: 'aE6IRWdETCdeOGjo',
    });

    await app.service('nfc-tags').create({ nfcId: '2589851589', audioUrl: 'spotify:track:2ej1A2Ze6P2EOW7KfIosZR' });

    await app.service('nfc-tags').create({ nfcId: '2589689541', audioUrl: 'spotify:track:4uLU6hMCjMI75M1A2tKUQC' });

    await app.service('nfc-tags').create({ nfcId: '2589928949', audioUrl: 'spotify:track:6tYlLMni6GwUksie3N6IPA' });
  } catch (error) {
    console.log('Seeding failed', (error as Error).message);
  }
}
```

Reply:
Good idea 😄